client_consumer/pulp/client/consumer/cli.py

```diff
@@ -178,7 +178,7 @@ def register(self, **kwargs):
 
         id_cert_name = self.context.config['filesystem']['id_cert_filename']
         cert_filename = os.path.join(id_cert_dir, id_cert_name)
-        fp = open(cert_filename, 'w')
+        fp = os.fdopen(os.open(cert_filename, os.O_WRONLY | os.O_CREAT, 0600), 'w')
```

Comment:
I'd love a little comment explaining what the flags do instead of having to "man 2 open"